templates/pages/brand.html

```diff
@@ -23,9 +22,9 @@
 <h1 class="page-heading">{{brand.name}}</h1>
 {{{snippet 'brand'}}}
 <div class="page">
-    <aside class="page-sidebar" id="faceted-search-container">
-        {{> components/brand/sidebar}}
-    </aside>
+    <!--<aside class="page-sidebar" id="faceted-search-container">-->
```

Comment:
just something to note, in some cases commenting out handlebars will still render :/